10/umbraco-cms/reference/cache/icacherefresher.md

```diff
@@ -9,8 +9,8 @@ _This section describes what ICacheRefresher and ICacheRefresher&lt;T&gt; are an
 
 ## What is an ICacheRefresher
 
-This interface has been in the Umbraco core for quite some time but has really only been used to ensure that content cache is refreshed among all server nodes participating in a load balanced scenario.
+This interface has been in the Umbraco core for a significant period. However, it has really only been used to ensure that content cache is refreshed among all server nodes participating in a load balanced scenario.
 
-With changes in 6.1, this has changed slightly, an `ICacheRefresher` is now the primary way to invalidate *any* cache that needs to be refreshed or removed regardless of whether we are in a load balanced environment.
+With changes in 6.1, this has changed slightly. An `ICacheRefresher` is now the primary method. It invalidates *any* cache needing refreshment or removal. This applies regardless of a load balanced environment.
```

Comment:
Seems this content has been here for a while as it is mentioning v6.1. In this case we should remove the version and only keep what is essential in this case.
```suggestion
An `ICacheRefresher` is the primary method that invalidates *any* cache needing refreshment or removal. This applies regardless of a load balanced environment.
```